tools/hardware/irq.py

```diff
@@ -185,6 +185,7 @@ def parse_irq(self, child, data):
     'qcom,msm-qgic2': ArmGic,
     'ti,am33xx-intc': RawIrqController,
     'ti,omap3-intc': RawIrqController,
+    'riscv,plic0': RawIrqController,
```

Comment:
We could make this a separate commit, because it is really separate from the other changes